images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -547,6 +547,11 @@ function Get-YqVersion {
     return "$yqVersion"
 }
 
+function Get-ImageMagickVersion {
+    $imagemagickVersion = Run-Command "magick -version | head -n 1 | cut -d" " -f 2,3"
```

Comment:
we have `Take-Part` function intended for such needs

Reply:
I have changed it so that it uses Take-Part instead